src/components/DropdownItem.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+    <div :style="{ paddingLeft: `${indentation * 1.6 + 0.5}rem` }"
+         @click="handleClick"
+         class="dropdown-item-div">
+        <template v-if="option.hasChildren">
+            <div class="icon-div" @click="handleIconClick">
+                <vue-feather type="chevron-right"
+                            v-show="!option.open"
+                            class="icon"/>
+                <vue-feather type="chevron-down"
+                            v-show="option.open"
+                            class="icon"/>
```

Comment:
so the reason i did this was because v-show will actually load both of them so if someone were to spam click, this performs slightly better, obviously not noticeably but thats why i tried to do it this way, happy to change it though! up to you really XD

Reply:
Ah ok - leave it is if it behaves more nicely this way!